src/redist/targets/Versions.targets

```diff
@@ -8,7 +8,7 @@
     <!--
         When DropSuffix is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <DropSuffix>false</DropSuffix>
+    <DropSuffix>true</DropSuffix>
```

Comment:
This should be reverted. It will make 3.1.200 builds stable.

Reply:
Reverting in #6179 (see [most recent commit](https://github.com/dotnet/core-sdk/pull/6179/commits/46b8ddf0b55ed52ffcf5eb8c4fc1d3ba66a4343e))